Get error with find parameters when trying to install v.5

**Describe the bug**
I try yo run `npm install gridjs --save` but get the following error

```shell
> gridjs@5.0.1 postinstall C:\Program Files\nodejs\node_modules\gridjs
> find . ! -path '*/node_modules/*' ! -path './package.json' ! -path './tests/dev-server/package.json' -name 'package.json' -execdir npm install \;

FIND: Parameter format not correct
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! gridjs@5.0.1 postinstall: `find . ! -path '*/node_modules/*' ! -path './package.json' ! -path './tests/dev-server/package.json' -name 'package.json' -execdir npm install \;`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the gridjs@5.0.1 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

**To Reproduce**
Not sure if it has anything to do with it, but I'm on node v14.15.0

**Expected behavior**
Would expect it to install. 
I ran `npm install gridjs@4.0.0 --save` and it installed that version without problems


**Desktop (please complete the following information):**
 - OS: Windows 10
 - Using windows terminal. Node version 14.15.0. npm version 6.14.8
 - Browser N/A
 - Version [N/A


**Additional context**
I ran `npm install gridjs@4.0.0 --save` and it installed that version without problem. So it looks like a problem with the latest version.

--- a/package.json
+++ b/package.json
@@ -99,7 +99,7 @@
     "typescript": "^4.2.4"
   },
   "scripts": {
-    "postinstall": "find . ! -path '*/node_modules/*' ! -path './package.json' ! -path './tests/dev-server/package.json' -name 'package.json' -execdir npm install \\;",
+    "prepare": "find . ! -path '*/node_modules/*' ! -path './package.json' ! -path './tests/dev-server/package.json' -name 'package.json' -execdir npm install \\;",
     "clean": "rimraf coverage dist tmp l10n/dist 'plugins/!(node_modules)/dist'",
     "prebuild": "npm run clean",
     "build": "run-p build:grid build:i18n build:plugins build:themes",

